common/device-model.cpp

```diff
@@ -3112,9 +3117,7 @@ namespace rs2
                             const ImVec2 pos = ImGui::GetCursorPos();
 
                             draw_later.push_back([windows_width, &window, sub, pos, &viewer, this, pb]() {
-                                if (!sub->streaming || !sub->post_processing_enabled) ImGui::SetCursorPos({ windows_width - 35, pos.y - 3 });
-                                else
-                                    ImGui::SetCursorPos({ windows_width - 35, pos.y - 3 });
+                                ImGui::SetCursorPos({ windows_width - 42, pos.y - 3 });
```

Comment:
A `if/else` statement was removed because there `if` scope was identical to `else` scope.

Reply:
OK